recipes/awherepy/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "awherepy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b7a1c5a09432f6c1870499bbde8f7ee33396794a9ee1df5196fcc0665bb971a3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - matplotlib-base
+    - numpy
+    - seaborn
+    - geopandas
+    - earthpy
+    - shapely
+    - rasterstats
+    - descartes
+    - rtree
+
+test:
+  imports:
+    - awherepy
+    - awherepy.agronomics
+    - awherepy.crops
+    - awherepy.fields
+    - awherepy.grids
+    - awherepy.plantings
+    - awherepy.models
+    - awherepy.weather
```

Could we add ``pip check`` here please?
```suggestion
    - awherepy.weather
  commands:
    - pip check
  requires:
    - pip
```

---

Copy that. Added `pip check`.